rerun_py/rerun_sdk/rerun/error_utils.py

```diff
@@ -205,7 +205,7 @@ def __exit__(
         exc_tb: TracebackType | None,
     ) -> bool:
         try:
-            if exc_type is not None and not strict_mode():
+            if exc_type is not None and exc_type is not KeyboardInterrupt and not strict_mode():
```

Comment:
I believe this is the better fix - this will ignore both `KeyboardInterrupt` and `SystemExit`



```suggestion
            if exc_type is not None and exc_type is Exception and not strict_mode():
```

Reply:
If that's the intention, perhaps worth adding a comment to the effect as well?

Reply:
@emilk well spotted (I had forgotten about that `Exception` vs. `BaseException` thing). I fixed the code with the correct `issubclass` check  + a comment.